source/devices/AM335X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -94,12 +96,11 @@ Build Information
 U-Boot
 -------------------------
```

Comment:
Can you handle the underline overrun 

(As you are handling it in `source/devices/AM437X/linux/Release_Specific_Release_Notes.rst`)